lib/Backup/Contacts.php

```diff
@@ -94,14 +94,14 @@ public function key()
 
     /**
      */
-    public function next()
+    public function next(): void
```

Comment:
Tentative return type

Reply:
I'm a bit concerned with suddenly adding  PHP 8.1 language feature. This could cause some serious headaches for anyone running on, say, a Ubuntu 20 LTS system without having to manually go through a PHP upgrade.  An argument could definitely be made for the fact that master is a development branch, but considering the length of time since our last release and the fact that a number of people are running some sort of "master checkout", I'd like to avoid this kind of "surprise" for people, especially when we can silence the return type warnings in a BC way (for now).